subprojects/test-runner/client/src/test/kotlin/com/avito/runner/scheduler/runner/RunnerIntegrationTest.kt

```diff
@@ -65,6 +69,8 @@ internal class RunnerIntegrationTest {
         minimumFailedCount = 0
     )
     private val loggerFactory = StubLoggerFactory
+    private val <E> Channel<E>.isClosedForSendAndReceive: Boolean
```

Comment:
Let's move it to place where it's used

Reply:
✅

Reply:
Moved this extension to `:common:coroutines-extension`. It seems to be the correct place for this property.